spk/synocli-disk/Makefile

```diff
@@ -0,0 +1,30 @@
+SPK_NAME = synocli-disk
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/synocli-disk.png
```

Comment:
I would also upload SVG file with copyright

Reply:
I added the svg files for all synocli.* packages from https://seby.io/files/ as mentioned [here](https://github.com/SynoCommunity/spksrc/issues/3254#issuecomment-421893202), but found no copyright entry within the files.